### DISCUSSION TITLE
关于校园邮箱注册的onedrive认证

### DISCUSSION THREAD

**Discussion by winsight** [spencerwooo/onedrive-vercel-index#631] [Q&A] [answered]
校园邮箱注册的onedrive，在“第二步 - 获取认证码”无法登陆，显示需要管理员授权。那么接下来注册自己的client_id 与 client_secret，是需要同样的校园邮箱，还是随意其他微软账号。

**Comment by winsight** [comment]
试了一下校园邮箱注册的Azure，登录之后无法注册 应用程序，显示无权限。

**Comment by myl7** [comment, marked as answer]
在组织中任何应用（或许除了 OneDrive 官方的一些应用）应该都需要管理员授权。
除了联系校方应该没有其他方法。

**Reply by winsight** [reply]
好的，明白了😭，谢谢啦

**Reply by myl7** [reply]
补充一下，此类 OneDrive 账户虽然无法授权应用，但是是可以使用 WebDAV 进行挂载的，有需要的话可以寻找一下支持 WebDAV 的类似应用

**Reply by tractortoby** [reply]
就是我这种情况是吧，那算了我不搞了http://localhost/?error=invalid_client&error_description=AADSTS650051%3A+Using+application+'onedrive-vercel-index'+is+currently+not+supported+for+your+organization+email.XXX.edu.cn+because+it+is+in+an+unmanaged+state.+An+administrator+needs+to+claim+ownership+of+the+company+by+DNS+validation+of+email.XXX.edu.cn+before+the+application+onedrive-vercel-index+can+be+provisioned.
Trace+ID%3A+05c04a95-0b85-4587-b2bd-28d1500f3700
Correlation+ID%3A+4e27afad-85d8-4e35-ac2c-47f7f7fabf9b
Timestamp%3A+2022-08-09+14%3A29%3A37Z#
